test/tests/test-xcm-para.ts

```diff
@@ -0,0 +1,730 @@
+		parachainApi,
+		sudoKeyring,
+		parachainApi.tx.sudo.sudo(
+			parachainApi.tx.assetManager.setAssetUnitsPerSecond(assetId, 0)
+		)
+	);
+	return { events, assetId };
+}
+
+describeParachain(
+	"XCM - receive_relay_asset_from_relay",
+	{ chain: "rococo-local" },
+	(context) => {
+		it.skip("should be able to receive an asset from relay", async function () {
+			const keyring = new Keyring({ type: "sr25519" });
+			const aliceRelay = keyring.addFromUri("//Alice");
+			const alicePara = keyring.addFromUri("//Alice"); // They are the same for the polkadot-launch tests
+
+			// const alith = await keyring.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+
+			const parachainOne = context.polkadotApiParaone;
+			const relayOne = context._polkadotApiRelaychains[0];
+
+			// Log events
+			logEvents(parachainOne, "PARA");
+
+			logEvents(relayOne, "RELAY");
+
+			await new Promise((res) => setTimeout(res, 10000));
+
+			// PARACHAINS
+			// registerAsset
+			const { events, assetId } = await registerAssetToParachain(
+				parachainOne,
+				alicePara
+			);
+
+			expect(events[0].toHuman().method).to.eq("UnitsPerSecondChanged");
+			expect(events[2].toHuman().method).to.eq("ExtrinsicSuccess");
+
+			// check asset in storage
+			const registeredAsset = await parachainOne.query.assets.asset(assetId);
+			expect((registeredAsset.toHuman() as { owner: string }).owner).to.eq(
+				palletId
+			);
+
+			// RELAYCHAIN
+			// Trigger the transfer
+			const { events: eventsRelay } = await createBlockWithExtrinsicParachain(
+				relayOne,
+				aliceRelay,
+				relayOne.tx.xcmPallet.reserveTransferAssets(
+					{
+						V1: {
+							parents: new BN(0),
+							interior: { X1: { Parachain: new BN(1000) } },
+						},
+					},
+					{
+						V1: {
+							parents: new BN(0),
+							interior: {
+								X1: {
+									AccountKey20: { network: "Any", key: alicePara.addressRaw },
+								},
+							},
+						},
+					},
+					{
+						V0: [
+							{
+								ConcreteFungible: {
+									id: "Here",
+									amount: new BN(THOUSAND_UNITS),
+								},
+							},
+						],
+					},
+					0,
+					new BN(4000000000)
+				)
+			);
+			expect(eventsRelay[0].toHuman().method).to.eq("Attempted");
+
+			// Wait for parachain block to have been emited
+			await waitOneBlock(parachainOne, 2);
+
+			// about 1k should have been substracted from AliceRelay
+			expect(
+				(
+					(await relayOne.query.system.account(aliceRelay.address)) as any
+				).data.free.toHuman()
+			).to.eq("8.9999 kUnit");
+			// Alith asset balance should have been increased to 1000*e12
+			expect(
+				(
+					await parachainOne.query.assets.account(assetId, alicePara.addressRaw)
+				).toHuman().balance === "1,000,000,000,000,000"
+			).to.eq(true);
+		});
+	}
+);
+
+// describeParachain("XCM - send_relay_asset_to_relay", { chain: "moonbase-local" }, (context) => {
+//   let keyring: Keyring,
+//     aliceRelay: KeyringPair,
+//     alith: KeyringPair,
```

Comment:
lets put this code into an issue to be fixed, and then remove it from this PR

Reply:
Might as well remove it altogether I dont think it adds value